mreg_cli/api/abstracts.py

```diff
@@ -400,6 +391,21 @@ def get_by_query_unique(cls, data: dict[str, str]) -> Self | None:
             return None
         return cls(**obj_dict)
 
+    @classmethod
+    def _get_by_location_path(cls, location: str) -> Self | None:
+        """Fetch an object by its location as returned by a `POST` request.
+
+        :param location: The location of the object.
+        :returns: The object if found, None otherwise.
+        """
+        # Some endpoints return invalid location headers,
+        # so we need to look up the object by other means.
+        if location.startswith(Endpoint.Labels):
+            # https://github.com/unioslo/mreg/blob/eed5c154bcc47b1dea474feabad46125ebde0aec/mreg/api/v1/views_labels.py#L30
+            # https://github.com/unioslo/mreg/blob/eed5c154bcc47b1dea474feabad46125ebde0aec/mreg/api/v1/views.py#L187
+            return cls.get_by_field("name", location.rpartition("/")[-1])
```

Comment:
When https://github.com/unioslo/mreg/issues/532 is fixed, this'll still be correct (as long as our assumptions are the same), but yeah. This is better for now.